scripts/sfpshow

```diff
@@ -408,24 +409,22 @@ class SFPShow(object):
         if self.intf_name is not None:
             presence = self.db.exists(self.db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(self.intf_name))
             if presence:
-                self.output = self.convert_interface_sfp_info_to_cli_output_string(
+                self.intf_eeprom[self.intf_name] = self.convert_interface_sfp_info_to_cli_output_string(
                     self.db, self.intf_name, self.dump_dom)
             else:
-                self.output += (self.intf_name + ': ' + 'SFP EEPROM Not detected' + '\n')
+                self.intf_eeprom[self.intf_name] = "SFP EEPROM Not detected\n"
         else:
             port_table_keys = self.db.keys(self.db.APPL_DB, "PORT_TABLE:*")
-            sorted_table_keys = natsorted(port_table_keys)
```

Comment:
Are you suggesting that we keep them sorted how they are, and instead display what ASIC they come from instead? (Since they are sorted with respect to ASIC instance)

Reply:
@nathcohe not suggesting. Just wanted to see the mapping of logical ports to ASIC namespace in your platform.

Reply:
@prgeor, our mapping is determined on a set of files that are specific to the platform. Our ASIC also takes different subsets of ports that may not be in order. That is, ASIC 0 might have `Ethernet0` and `Ethernet2` and ASIC 1 might have `Ethernet1`, and so on.

Reply:
@prgeor I can get you a specific mapping example if you would like.